cmake/BLTOptions.cmake

```diff
@@ -53,7 +53,6 @@ option(ENABLE_COVERAGE   "Enables code coverage support" OFF)
 ################################
 # Build Options
 ################################
-option(ENABLE_COPY_HEADERS "Copy headers to build directory" OFF)
```

Comment:
I appreciate the simplicity that comes from ripping this feature out, but do we have workarounds for this feature in all codes that were using ``COPY_HEADERS``?

Should we check that these workarounds work (e.g. by disabling ``COPY_HEADERS``) before removing the feature from blt?

Reply:
AXOM is the only code I know that uses copy headers.

Reply:
I meant that we should have a working solution there with ``ENABLE_COPY_HEADERS`` disabled before we remove it from blt.

Reply:
@kennyweiss, I think we agree any `COPY_HEADERS` solution will be in Axom proper, so I am not sure that should hold up work on BLT master.